### PR TITLE
Editorial cleaning

### PR DESCRIPTION
**Pull request by OR13**
Mostly removing unused references, one instance of removing a vendor specific technology reference (IPFS), and one instance of removing an expired IETF I-D "HASHLINKS".

***

### :boom: Error: 500 Internal Server Error :boom: ###

[PR Preview](https://github.com/tobie/pr-preview#pr-preview) failed to build. _(Last tried on Oct 18, 2023, 3:04 PM UTC)_.

<details>
<summary>More</summary>

PR Preview relies on a number of web services to run. There seems to be an issue with the following one:

:rotating_light: [Spec Generator](https://www.w3.org/2015/labs/) - Spec Generator is the web service used to build specs that rely on ReSpec.

:link: [Related URL](https://labs.w3.org/spec-generator/?type=respec&url=https%3A%2F%2Fraw.githubusercontent.com%2Fw3c%2Fvc-data-model%2F15eb092a1107dd7ed5bbc60a90c502609734d82d%2Findex.html%3FisPreview%3Dtrue)

```

😭  Sorry, there was an error generating the HTML. Please report this issue!
Specification: http://labs.w3.org/spec-generator/uploads/XNcST4/index.html?isPreview=true%3FisPreview%3Dtrue&publishDate=2023-10-18
ReSpec version: 34.2.0
File a bug: https://github.com/w3c/respec/
Error: undefined

```

_If you don't have enough information above to solve the error by yourself (or to understand to which web service the error is related to, if any), please [file an issue](https://github.com/tobie/pr-preview/issues/new?title=Error%20not%20surfaced%20properly&body=See%20w3c/vc-data-model%231304.)._
</details>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5186,7 +5182,7 @@ <h3>Content Integrity Protection</h3>
   "type": ["VerifiableCredential", "ExampleAlumniCredential"],
   "credentialSubject": {
     "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
-    "image": <span class="highlight">"ipfs:/ipfs/QmXfrS3pHerg44zzK6QKQj6JDk8H6cMtQS7pdXbohwNQfK/image"</span>,
+    "image": <span class="highlight">"https://example.com/image"</span>,
```
Comment (OR13) [inline index.html:5185]: similar to not including direct references to bitcoin or ethereum, IPFS is a product from a specific W3C member... we should avoid the appearance of bias.

**Comment by OR13** [comment]
See related PR in VC Specs DIR: https://github.com/w3c/vc-specs-dir/pull/29#pullrequestreview-1658575758

File: index.html (hunk 1 above)
Comment (OR13) [inline index.html:5185]:
```suggestion
    "image": <span class="highlight">"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVR42mNk+M9QDwADhgGAWjR9awAAAABJRU5ErkJggg=="</span>,
```

Comment (OR13) [reply index.html:5185]: ^ in case folks prefer to not encourage tracking via HTTPS urls, I provide the following small, but valid data uri.

**Review by decentralgabe — APPROVED** [review]

**Review by brentzundel — APPROVED** [review]

**Review by msporny — APPROVED** [review]

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2023-10-11-vcwg#section1-5) on 2023-10-11

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 1.5. Editorial cleaning (pr vc-data-model#1304)

_See github pull request [vc-data-model#1304](https://github.com/w3c/vc-data-model/pull/1304)._




**Brent Zundel:** This is some editorial clean up submitted by Orie, a number of approvals already, fixing references.  
… I expect this to just be merged relatively quickly, happy to take comments if any.  

</details>

**Comment by msporny** [comment]
@OR13 trying to merge, but conflicts exist now in common.js, please rebase and resolve the conflicts so this PR can be merged.

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2023-10-17-vcwg-special#section2-10) on 2023-10-17

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 2.10. Editorial cleaning (pr vc-data-model#1304)

_See github pull request [vc-data-model#1304](https://github.com/w3c/vc-data-model/pull/1304)._




**Brent Zundel:** I think this PR just needs to be merged but has conflicts.  
… Orie needs to fix the conflicts.  

</details>

**Comment by OR13** [comment]
I resolved the conflict via the web ui, but I need to confirm the behavior locally before this can be merged

**Comment by OR13** [comment]
Screen Shot 2023-10-18 at 10 00 37 AM

W3C outage is making it tough to confirm that this is working.

**Comment by msporny** [comment]
Editorial, multiple reviews, no changes requested, no objections, merging.

**Pull request merged**